src/encoded/search_views.py

```diff
@@ -711,3 +714,94 @@ def top_hits(context, request):
         ]
     )
     return fr.render()
+
+
+@view_config(route_name='rnaget', request_method='GET', permission='search')
```

Comment:
Can we move logic to its own module/file and just expose entry point here? Will allow for cleaner development in the future if we have encapsulated space for helper functions/tests and can modify functionality without editing this file.

Reply:
yes, this was on my to-do list, it just got de-prioritized as I was rushing to get this into this sprint. I will push a refactor